services/player-service/pom.xml

```diff
@@ -130,6 +131,11 @@
                 <artifactId>spring-boot-starter</artifactId>
                 <version>${spring-boot-starter.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-data-mongodb</artifactId>
```

Comment:
I think you only need this on the modules that need to use mongodb, it should not be in root pom. Now the version definition should be yes